http-server/src/main/java/org/triplea/server/error/reporting/ErrorReportingDao.java

```diff
@@ -0,0 +1,46 @@
+package org.triplea.server.error.reporting;
+
+import java.time.Instant;
+
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+/**
+ * DAO class for error reporting functionality.
+ */
+public interface ErrorReportingDao {
```

Comment:
This class uses JDBI, SQL Object syntax. 
Overall it seems pretty nice!

@ron-murhammer FYI, you may be interested to see this example. I was tempted to resurrect the conversation to use MyBatis and go for that. I was reading through drop wizard docs on how to configure DB and they strongly recommended to use JDBI (Sql Object). Specifically we're using JDBI3 here.

Reply:
Of note also, database-rider is agnostic to what framework we use, it was pretty straight forward to test this code using DB rider. The DAO pattern also made it easy to mock the DB interaction and test the classes that used this DAO.